ValveResourceFormat/IO/MapExtract.cs

```diff
@@ -359,10 +355,79 @@ public string ToValveMap()
             }
         }
 
+        //convert phys to hammer meshes
+        var phys = LoadWorldPhysics();
+        if (phys != null)
+        {
+            foreach (var hammermesh in PhysToHammerMeshes(phys))
+            {
+                MapDocument.World.Children.Add(new CMapMesh() { MeshData = hammermesh });
+            }
+        }
+
```

Comment:
This patch confuses me. We (used to) also export a physics model. Where has that been changed to remove meshes that are now handled by this?